### PR TITLE
Created homepage with hard-coded search bar and gallery.

### PR DESCRIPTION
**Pull request by TengkuHanaShafikah**
Below is a preview of the homepage. If time allows, I can add a small "What is TravelBud?" section below the Featured Destinations gallery. Let me know if there is anything that you would like for me to add/edit!

TravelBud Homepage

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,169 @@
...
+@import url('https://fonts.googleapis.com/css?family=Sriracha');
+
+/*--------------
+Site Name
+---------------*/
+#site-name {
+  color: #00bcd4;
+  font-family: 'Sriracha', Helvetica, Arial, sans-serif;
+  font-size: 40px;
+  font-weight: 400;
```
Comment (josueetcom) [inline src/main/webapp/style.css]: [Style](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties): Use shorthand properties where possible.

Here and below you can simply use the `font` property to set these values

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,169 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/*--------------
+Imported Fonts
+---------------*/
+@import url('https://fonts.googleapis.com/css?family=Montserrat:100,200,300,400,500,600,700,800,900');
+@import url('https://fonts.googleapis.com/css?family=Sriracha');
```
Comment (josueetcom) [inline src/main/webapp/style.css]: In order to get the checks to pass I think you can instead import these via `<link>` tags in the HTML:
https://developers.google.com/fonts/docs/getting_started#a_quick_example

**Review by josueetcom — DISMISSED** [review]
Nothing major, LGTM!

**Review by gferioli0418 — DISMISSED** [review]
LGTM!

**Review by josueetcom — APPROVED** [review]
LGTM

**Pull request merged**